mlflow/tracking/fluent.py

```diff
@@ -526,6 +526,8 @@ def active_run() -> Optional[ActiveRun]:
     """
     Get the currently active ``Run``, or None if no such run exists.
 
+    **Note**: The API is thread-local.
```

Comment:
```suggestion
    .. attention::

        This API is **thread-local** and returns only the active run in the current thread. If your application is multi-threaded and a run is started in a different thread, this API will not retrieve that run.
```

"thread-local" alone is not clear enough for users especially data scientists

Reply:
Can we add this to `last_active_run` too